src/components/Buttons/Button.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  onClick: () => void;
+  type?: 'button' | 'submit' | 'reset';
+  className?: string;
```

Comment:
```suggestion
```

Würde ich eher weglassen. Soweit ich sehe, gibt es bisher keine UseCase. Und es ist unüblich sowohl small|large|outline probs zu haben als auch classNames zu übergeben. Es lässt die Frage offen, wie die component classes vs prop classes zueinander stehen. Ersetzen prop component? Ergänzen sie? Was ist bei Dopplungen oder Überschreibungen?… 

Es gibt ein TW-React-Package das dieses Thema behandeln, aber ich finde es als Dependency zu "gefährlich".

Reply:
Wie sieht es aus mit margins usw.? Dann müsste man immer einen Container darüber machen.

Reply:
I just copied this from the [`ButtonLink`](https://github.com/FixMyBerlin/fixmy.rsv/blob/develop/src/components/Links/ButtonLink.tsx) component. Removing the `className` prop would require additional DOM-elements  for further styling, as @ohrie already mentioned.